geometry_msgs/Vector3[] data # array of Vector3